_inc/client/components/admin-notices/index.jsx

```diff
@@ -7,11 +7,16 @@ const AdminNotices = React.createClass( {
 		let $vpNotice = jQuery( '.vp-notice' );
 		if ( $vpNotice.length > 0 ) {
 			$vpNotice.each( function () {
+				if ( jQuery( this ).children( '.vp-message' ).children( 'p' ).html().includes( 'VP08' ) ) {
+					jQuery( this ).children( '.vp-message' ).children( 'p' ).html( 'Your VaultPress subscription is no longer active. Please confirm the status of your account. <a href="https://dashboard.vaultpress.com/account/">View account</a>' );
```

Comment:
Would it be possible to pull the string from VaultPress instead, so it can be translated?